contrib/cdisetup/nvidia/nvidia.go

```diff
@@ -39,10 +39,6 @@ type setup struct{}
 var _ cdidevices.Setup = &setup{}
 
 func (s *setup) Validate() error {
-	_, err := readVersion()
-	if err == nil {
-		return nil
-	}
 	b, err := hasNvidiaDevices()
```

Comment:
This would still make the device show up if kernel driver is missing but otherwise GPU is available. Is this what we want? Ideally we would show clear error about missing driver in this case to the user but not sure it is possible atm before actually triggering a build.